modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc

```diff
@@ -19,12 +19,7 @@ When you create the Connected App to manage client applications, ensure that you
 
 . Select *App acts on its own behalf (client credentials)* for *Type*.
 . Add the *Application Creator* scope.
-
-image::connected-app-client-apps.png[Screenshot calling out the location of App acts on its own behalf (client credentials) and Application Creator]
-
-[calloutlist]
-. *App acts on its own behalf (client credentials)* type
-. *Add Scopes* button to add *Application Creator* scope
+. Select the root organization as the *Business Group* as you add this scope.
```

Comment:
The "as...as" structure can be ambiguous. What's the meaning of this sentence?:

-  Select the root organization as the *Business Group* when you add this scope.
-  Select the root organization as the *Business Group* if you add this scope.
-  Select *Business Group* in the root organization when/if you add this scope.

Reply:
Good point! Now reworded to be more clear :)